mmv1/products/storage/api.yaml

```diff
@@ -415,6 +415,14 @@ objects:
             name: 'enabled'
             description: |
               While set to true, versioning is fully enabled for this bucket.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'autoclass'
+        description: "The bucket's autoclass configuration." 
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'enabled'
+            description: |
+              While set to true, utoclass automatically transitions objects in your bucket to appropriate storage classes based on each object's access pattern.  
```

Comment:
It looks like this resource is handwritten, and only your changes to `resource_storage_bucket.go.erb` are being used. Do you know if the changes here to `api.yaml` are needed?

I suspect that this is a case where the `api.yaml` config has been left in the repo but should be removed in favor of the handwritten `resource_storage_bucket.go.erb`. If that's the case, for now we should just remove these additions from the changeset.